imageroot/events/mail-settings-changed/80start_services

```diff
@@ -13,5 +13,16 @@ import os
 event = json.load(sys.stdin)
 
 if event['module_uuid'] == os.getenv('MAIL_SERVER', ''):
-   # Restart to apply changed settings
+   # Restart to apply changed settings but first retrieve the IP of the first provider
+
+   #Connect to redis
+   rdb = agent.redis_connect()
```

Comment:
we have just received the event from redis so we are connected to redis, my guess is since we have just received the event , does the replica is up to date ???

well not sure here

Reply:
done